cache/sharded_cache.h

```diff
@@ -27,8 +27,11 @@ class CacheShard {
   virtual Status Insert(const Slice& key, uint32_t hash, void* value,
                         size_t charge,
                         void (*deleter)(const Slice& key, void* value),
-                        Cache::Handle** handle, Cache::Priority priority) = 0;
-  virtual Cache::Handle* Lookup(const Slice& key, uint32_t hash) = 0;
+                        Cache::Handle** handle, Cache::Priority priority,
+                        const Slice& (*unpack)(void* value) = nullptr,
```

Comment:
On second thought, we should not use default argument for virtual functions, which is also mentioned explicitly in https://google.github.io/styleguide/cppguide.html#Default_Arguments.

Reply:
Gotcha.

Reply:
Well, `Cache::Insert()` already has `handle = nullptr` and `priority=LOW` as default argument. I do think this makes sense though; I'll tentatively get rid of the default values and re-order them before `handle`.

Reply:
> Well, `Cache::Insert()` already has `handle = nullptr` and `priority=LOW` as default argument. I do think this makes sense though; I'll tentatively get rid of the default values and re-order them before `handle`.

This is giving me a hell of troubles -- multiple modules originated from `Cache` are complaining about the ordering of arguments. I'll deal with this during wrap-up, I guess.